docs/nodes/bridge-node.md

```diff
@@ -150,7 +150,7 @@ Celestia Bridge Nodes
 Example:
 
 ```sh
-/ip4/46.101.22.123/tcp/2121/p2p/12D3KooWD5wCBJXKQuDjhXFjTFMrZoysGVLtVht5hMoVbSLCbV22
+/ip4/<ip-address>/tcp/2121/p2p/12D3KooWD5wCBJXKQuDjhXFjTFMrZoysGVLtVht5hMoVbSLCbV22
```

Comment:
localhost? Or put it in a shell variable

Reply:
@adlerjohn was talking to @renaynay who has a preference of not putting local-host in most places over using <ip-address> instead.

Can go either way, just providing context

Reply:
Okay, then shell variable would be preferred

Reply:
check latest commit and lemme know if that's what you want